packages/teams-js/src/public/pages.ts

```diff
@@ -199,9 +184,9 @@ export namespace pages {
         throw errorNotSupportedOnPlatform;
       }
       if (runtime.isLegacyTeams) {
-        resolve(sendAndHandleStatusAndReason('executeDeepLink', createTeamsAppLink(params)));
+        resolve(sendAndHandleStatusAndReasonWithVersion(ApiVersion.V_3, 'executeDeepLink', createTeamsAppLink(params)));
```

Comment:
nit: How about V_21 instead of V3, it is a call from the version 2 but running in legacy teams (which is old in some sense). I think V3 should be reserved for future revisions rather than address peculiar flow.

Reply:
Not a nit. It is a unique 1-1 matching and 21 will mess the order of numbering.